features/wheel-events.yml

```diff
@@ -0,0 +1,14 @@
+name: Wheel events
+description: Wheel events occur when the user moves a mouse wheel or similar spatially rotating input device.
+spec: https://w3c.github.io/uievents/#events-wheelevents
+status:
+  compute_from: api.WheelEvent
```

Comment:
Pin to the `_event` keys. If the event doesn't actually fire, there's no sense reporting the status of the interface.